sql/sql_table.cc

```diff
@@ -9790,6 +9790,8 @@ bool mysql_checksum_table(THD *thd, TABLE_LIST *tables,
 	      row_crc= my_checksum(row_crc, t->record[0], t->s->null_bytes);
             }
 
+	    uchar *checksum_start= NULL;
+	    size_t checksum_length= 0;
```

Comment:
Thanks, though I didn't mean you need to fix the whole function. :) Usually we only make added/modified lines to confirm to coding style, disregarding context.